tests/api.c

```diff
@@ -20109,6 +20109,200 @@ static void test_wolfSSL_DES_ncbc(void){
 #endif
 }
 
+static void test_wolfSSL_AES_cbc_encrypt()
```

Comment:
General note: The `wc_AesCbcEncrypt` function allows a size that is not a multiple of block size, but will simply ignore the remainder without failure. The `wc_AesCbcDecrypt` function will error if not a block size multiple. It might be worth discussing the reasons for `wc_AesCbcEncrypt` silently ignoring bytes when not block size aligned.

Reply:
Excellent note. Yes we should go over that!